### ISSUE TITLE
Add Storybook

### ISSUE THREAD

**Issue by mehranmf31** [aliehyaie/React-Design#80]
I think it's better to add storybook to create a better documentation and remove these large tables from readme.

**Comment by aliehyaie** [comment]
I agree with you, and I will do it sometime in the near future. @mehranmf31
